utils/extractor/src/index.ts

```diff
@@ -100,24 +98,24 @@ export const Extractor = (
             (acc: number, sheet: any) => acc + (sheet?.data?.length || 0),
             0
           )
+
+          await new Promise((resolve) => {
+            setTimeout(resolve, 3_000)
```

Comment:
Should we pull this `3_000` variable out into a config `option`?

Reply:
good thinking